libs/sdk-ui-charts/src/highcharts/adapter/plugins/autohideLabels/autohideColumnLabels.ts

```diff
@@ -259,15 +267,21 @@ function toggleStackedLabelsForSingleAxis() {
         const areOverlapping = areNeighborsOverlapping(neighbors);
 
         if (areOverlapping) {
-            this.userOptions.stackLabelsVisibility = "hidden";
-            stackTotalGroup.hide();
+            toggleStackTotalGroup(stackTotalGroup, false);
```

Comment:
Similar to previous, we could use `toggleStackTotalGroup(stackTotalGroup, !areOverlapping);`